charts/ocis/values.yaml

```diff
@@ -231,6 +231,21 @@ features:
         filter:
         # -- The object class to use for groups in the default group search filter like `groupOfNames`.
         objectClass: groupOfNames
+  # -- Define quota settings.
+  quotas:
+    # -- Sets the maximum quota for spaces
+    max: 
+    # -- Sets the default quota for spaces
+    default: 
+    # -- Sets specific quotas for roles
+    roles: {}
+    # roles:
+    #   # User Role set to 2GB
```

Comment:
While I added the unit description to `max` and `default` I've seen that the description might not be completely correct here. I copied the unit over, but 2GB is not exactly 2000000000 but more 2147483648. Depending if it's interpreted decimally or binary. How to you usually handle that. I've see different interpretations at various places in software so I would have this at least aligned in our documentation.

Reply:
Done with https://github.com/owncloud/ocis-charts/pull/159/commits/ddd30b4b818db28120ab793678ed13c7d2aaf901